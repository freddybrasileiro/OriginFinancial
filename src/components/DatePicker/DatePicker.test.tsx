import * as React from 'react';
import { shallow } from 'enzyme';
import { DatePicker } from './index';

describe('DatePicker', () => {
  it('Should render the DatePicker component', () => {
    expect(
      shallow(
        <DatePicker
          label="My date picker"
          value={[4, 2020]}
          onChange={() => {}}
        />
      )
    ).toMatchSnapshot();
  });
});
