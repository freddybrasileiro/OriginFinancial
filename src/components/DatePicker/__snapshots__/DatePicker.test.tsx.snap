// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker Should render the DatePicker component 1`] = `ShallowWrapper {}`;
